### DISCUSSION TITLE
Missing translation file for source code. - Joomla 5.0.1

### DISCUSSION THREAD

**Discussion by Magnytu2** [joomla/joomla-cms#47056] [2. Cold Features]
When using TinyMCE we can switch to the source code. Thank you, it is now possible to search with "CTRL F". But I think that one or more translation strings are missing for the buttons and functions.
Thanks for your help.

<img width="1784" alt="Window_et_Articles___Ajouter_-_Association_de_Recherche_en_Soins_Infirmiers_-_Administration" src="https://github.com/joomla/joomla-cms/assets/26197929/7000e1fb-76f5-4ef9-9b63-7c2ccfbd0399">

**Comment by ghost** [comment]
Isn't this done at Crowdin?

**Comment by Hackwar** [comment]
These buttons are handled by TinyMCE and are not translated by us. if you want this translated, please contact TinyMCE to get them to handle this here: https://github.com/tinymce/tinymce/issues Thank you.

Since this is not something in our power, I'm going to close this ticket. If you feel this is not correct, please respond here.

**Comment by brianteeman** [comment]
That is not correct. It is NOT from the tinymce repo
This is our code from 
`build\media_source\plg_editors_tinymce\js\plugins\joomla-highlighter\plugin.es6.js`

**Comment by Hackwar** [comment]
Ok, if you say so. I was under the impression that this is the tinymce sourcecode view. Reopening the issue.

**Comment by brianteeman** [comment]
its not what I say. It is what the code says

**Comment by rdeutz** [comment]
As far as I see it then the find is handeled by codemirror. There is a way of translating the keys, see https://codemirror.net/examples/translate/ but I am not sure if that is worth the effort. Will discuss it with the maintainers team.

**Comment by rdeutz** [comment]
We discussed this yesterday in the weekly maintainers meeting. We want to have as much translated as we can but we don't have this as a prio 1 on our list. If someone comes up with a solution/PR we are happy to look at.

**Comment by brianteeman** [comment]
Please note that this is NOT only about the use of codemirror to view the source code when in tinymce as in the initial report. The same is true wherever codemirror is used egc in the template manager

![image](https://github.com/joomla/joomla-cms/assets/1296369/a7ca0470-9a28-42d4-a7bd-5bd4db2bc9be)

**Comment by brianteeman** [comment]
See https://codemirror.net/examples/translate/ for how to translate codemirror
